Ruby/rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] It would be better if we have a belongs to relationship between Rental and a Book, Rental and a Person